crates/store/src/db/models/queries.rs

```diff
@@ -481,11 +481,20 @@ pub(crate) fn upsert_accounts(
         account_id: AccountId,
     ) -> Result<Vec<Account>, DatabaseError> {
```

Comment:
Leaving a comment here but it is actually for the line 472 above: I'm not sure that statement is accurate. I think this function should work fine when `details` are `NULL`. Otherwise, we have a bug, I think.